jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -56,9 +56,12 @@ def _report_failure(error_type: str, msg: str,
                       devs: Sequence[str] = all_devices) -> None:
     limitations.append(Limitation(prim.name, error_type, msg, tuple(devs)))
 
-  def tf_unimpl(np_dtype: NpDType, additional_msg: Optional[str] = None,
+  def tf_unimpl(np_dtype: Optional[NpDType] = None,
+                additional_msg: Optional[str] = None,
                 devs: Sequence[str] = all_devices) -> None:
-    msg = f"{prim.name} is unimplemented for dtype {np_dtype}"
+    msg = f"{prim.name} is unimplemented"
+    if np_dtype is not None:
+      msg += f" for dtype {np_dtype}"
     if additional_msg:
       msg += '; ' + additional_msg
     _report_failure("Missing TF support", msg, devs=devs)
```

Comment:
This change is in concurrence with a change I'm proposing in #4222. Which one do you think we should go for? A mix of the 2 is also possible :)

Reply:
If that PR is ready, we can merge it and I adapt. Otherwise, we merge this and you can adapt. I think that we will iterate on this quite a bit as we expand the categorization. Most likely there will be several helper functions, as in your PR.

Reply:
It is not ready. Let's merge this one, and I will adapt.